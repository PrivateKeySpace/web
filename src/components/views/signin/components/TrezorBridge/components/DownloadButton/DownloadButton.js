import React from 'react'
import { css } from 'aphrodite/no-important'
import tx from '../../../../../../../theme/styles'
import sx from './styles'

function DownloadButton () {
  return (
    <button className={css(tx.p0, tx.ml1, tx.textTransformUppercase, sx.button)}>Download</button>
  )
}

export default DownloadButton
