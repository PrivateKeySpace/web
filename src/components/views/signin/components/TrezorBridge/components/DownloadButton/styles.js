import { StyleSheet } from 'aphrodite/no-important'
import { colors, fontFamilies } from '../../../../../../../theme/constants'

const styles = StyleSheet.create({
  button: {
    backgroundColor: colors.darkBg,
    border: 'none',
    borderRadius: 3,
    color: colors.white,
    cursor: 'pointer',
    fontFamily: fontFamilies.primary,
    fontSize: 13,
    height: 32,
    width: 130
  }
})

export default styles
