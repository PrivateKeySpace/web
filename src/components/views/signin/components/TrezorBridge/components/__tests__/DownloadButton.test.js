import React from 'react'
import { StyleSheetTestUtils } from 'aphrodite'
import { render } from 'enzyme'
import DownloadButton from '../DownloadButton'

beforeEach(() => {
  StyleSheetTestUtils.suppressStyleInjection()
})

afterEach(() => {
  StyleSheetTestUtils.clearBufferAndResumeStyleInjection()
})

it('renders without crashing and matches snapshot', () => {
  const layout = render(
    <DownloadButton />
  )
  expect(layout).toMatchSnapshot()
})
