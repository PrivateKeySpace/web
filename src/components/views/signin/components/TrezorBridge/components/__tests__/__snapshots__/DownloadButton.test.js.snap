// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing and matches snapshot 1`] = `
<button
  class="p0_3js4c0-o_O-ml1_1dn7v29-o_O-textTransformUppercase_4efw5a-o_O-button_14du48h"
>
  Download
</button>
`;
